site/guides/consumer/sidebar.md

```diff
@@ -0,0 +1,30 @@
+- [Installation](installation/)
+- [Guides](guides/)
```

Comment:
Do you have to repeat all other sections every time? If we're adding a section, how many `md` files need to be udpated?

Reply:
> Is there a preview of the latest change, `https://kpt-dev-ee2a9.web.app/guides/` seems stale.

https://kpt-dev-ee2a9--migration-ep4dvb2t.web.app/ will be the hosted link for the preview of this PR

Reply:
There are eight distinct sidebar files, but only markdown files representing the immediate parent sidebar of the added section need to be updated (i.e., the `guides/consumer/sidebar.md` for any file added underneath the `consumer` direectory`). The only time all eight files need to updated is when there's a change to the top-level structure of Installation, Guides, etc.